index.js

```diff
@@ -0,0 +1,63 @@
+// Make sure if the local storage in empty than add an empty array
+if (localStorage.getItem('Added Books') == null) {
+  localStorage.setItem('Added Books', JSON.stringify([]));
```

Comment:
- [ ] At the reload of the webpage all of the data stored in the local storage are no longer visible as `Added books` shows up as an empty array with no values. The major reason for saving data in the localstorage is so that when a user reloads the page or leaves the browser whatever information previously saved is stored and displayed when the user revisits the page. Please kindly implement this properly so that the saved data is persisted upon reload of the webpage :+1: :sparkles: